dry_rest_permissions/generics.py

```diff
@@ -72,6 +73,8 @@ class DRYPermissions(permissions.BasePermission):
     2) Object permissions for a specific object (format has_object_{action}_permission):
         2a) specific action permissions (e.g. has_object_retrieve_permission)
         2b) general action permissions  (e.g. has_object_read_permission)
+    3) Django permissions (using user.has_perm method)
+        https://docs.djangoproject.com/en/3.1/topics/auth/default/#default-permissions
```

Comment:
The link has a warning " This document is for an insecure version of Django that is no longer supported. Please upgrade to a newer release! "